local-flake/flake.nix

```diff
@@ -58,6 +58,30 @@
           ldflags = ["-s" "-w" "-X main.version=${version}"];
         };
 
+        cloud-mta-build-tool = nixpkgsFor.${system}.buildGoModule rec {
```

Comment:
Shall we create an issue on their repo and request that this should be published as nix-package? For example like this here https://github.com/cloudfoundry/bosh-bootloader/issues/596. 

Even when we don't get it right away, they are at least aware of it that there is a demand for it.